hieradata/vagrant/common.yaml

```diff
@@ -333,6 +333,28 @@ profile::jenkinscontroller::jcasc:
           virtualNetworkResourceGroupName: "prod-jenkins-public-prod"
           subnetName: "ci.j-agents-vm"
           spot: true
+        - name: "win-2019-datadog-test" # The name must not contains "windows" or Azure API complains :facepalm:
+          description: "Windows 2019 test datadog"
+          imageDefinition: jenkins-agent-windows-2019
+          os: "windows"
+          storageAccount: SuperSecretThatShouldBeEncryptedInProduction
+          location: "East US 2"
+          instanceType: Standard_D4s_v3 # 4 vCPUS / 16 Gb
+          architecture: amd64
+          labels:
+            - docker-windows-test-datadog
+          idleTerminationMinutes: 30 # Quick deallocation as Linux is billed per minute
```

Comment:
```suggestion
```

Remove it here, and also on the `hieradata/clients/azure.ci.jenkins.io.yaml`  as per https://github.com/jenkins-infra/jenkins-infra/pull/2401 (a PR we merged during your night so you were not aware of it)